packages/components/src/sites-table/index.tsx

```diff
@@ -11,31 +14,85 @@ interface SitesTableProps {
 const Table = styled.table`
 	border-collapse: collapse;
 	table-layout: fixed;
+	@media only screen and ( max-width: 781px ) {
+		.sites-table__mobile-hidden {
+			display: none;
+		}
+	}
 `;
 
 const Row = styled.tr`
 	line-height: 2em;
 	border-bottom: 1px solid #eee;
+	td {
+		padding-top: 5px;
+		padding-bottom: 5px;
+		vertical-align: middle;
+	}
 `;
 
+const SiteName = styled.h2`
+	font-weight: 500;
+	font-size: 16px;
+	letter-spacing: -0.4px;
+	color: var( --studio-gray-100 );
+	a {
+		color: inherit;
+		&:hover {
+			text-decoration: underline;
+		}
+	}
+`;
+
+const SiteUrl = styled.a`
+	font-size: 14px;
+	line-height: 20px;
+	letter-spacing: -0.24px;
+	color: var( --studio-gray-60 );
+`;
+
+const displaySiteUrl = ( siteUrl: string ) => {
+	return siteUrl.replace( 'https://', '' ).replace( 'http://', '' );
+};
+
 export function SitesTable( { buildSiteUrl, className, sites }: SitesTableProps ) {
 	const { __ } = useI18n();
 
 	return (
 		<Table className={ className }>
-			<thead>
+			<thead className="sites-table__mobile-hidden">
 				<Row>
 					<th>{ __( 'Site', __i18n_text_domain__ ) }</th>
 					<th>{ __( 'Plan', __i18n_text_domain__ ) }</th>
+					<th>{ __( 'Last Publish', __i18n_text_domain__ ) }</th>
+					<th style={ { width: '20px' } }></th>
 				</Row>
 			</thead>
 			<tbody>
 				{ sites.map( ( site ) => (
 					<Row key={ site.ID }>
 						<td>
-							<a href={ buildSiteUrl ? buildSiteUrl( site ) : site.URL }>{ site.name }</a>
+							<div style={ { display: 'flex' } }>
+								<SiteIcon site={ site } size={ 50 } />
```

Comment:
Just wondering If would be better to add a placeholder component, say thumbnail that already has the defined dimensions based on the UI with a gray background until we start fetching thumbnail images.

Reply:
probably not better, but just to make it more consistent with UI screenshots provided.